### ISSUE TITLE
EditorAccessibility: items in {messages} array need key property

### ISSUE THREAD

**Issue by omnikrll** [processing/p5.js-web-editor#62]
The elements get painted to the DOM properly, but the following shows up in the console:

"Warning: Each child in an array or iterator should have a unique "key" prop. Check the render method of `EditorAccessibility`. See https://fb.me/react-warning-keys for more information.
    in p (created by EditorAccessibility)"
